front/src/reducers/osrdconf/operationalStudiesConf/index.ts

```diff
@@ -34,7 +33,7 @@ export const operationalStudiesConfSlice = createSlice({
 
       state.rollingStockID = rollingStock?.id;
       state.pathSteps = computeBasePathSteps(action.payload);
-      state.startTime = convertIsoUtcToLocalTime(start_time);
+      state.startTime = new Date(start_time);
```

Comment:
Instead of retransforming the start_time here, you can directly use startTime (which is already a Date ;) )

Reply:
I was worried about changing behavior in a refactoring, but it seems like this should be safe. Done!